packages/eslint-config-shakacode/scripts/release

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
Yep, I checked it out and it has bunch of settings to learn. Most of the time I need to just release patch or minor and most of the time this little shell script do what I need, so not sure it's worth it.